test_framework/musig.py

```diff
@@ -52,26 +52,15 @@ def sign_musig(priv_key, k_key, R_musig, P_musig, msg):
     """Construct a musig signature."""
     assert priv_key.valid
     assert priv_key.compressed
+    assert P_musig.compressed
     assert len(msg) == 32
     assert k_key is not None and k_key.secret != 0
-    Rm = SECP256K1.affine(R_musig.p)
-    assert jacobi_symbol(Rm[1], SECP256K1_FIELD_SIZE) == 1
-    R = SECP256K1.affine(SECP256K1.mul([(SECP256K1_G, k_key.secret)]))
```

Comment:
`SECP256K1_G` is no longer used. You can remove the import.

Reply:
Done, thanks!